src/canmatrix/log.py

```diff
@@ -27,8 +27,8 @@
 import logging
 
 
-def setup_logger(name):
-    """Setup a project wide logger singleton"""
+def setup_logger(name='canmatrix'):
```

Comment:
I would have expected `name` to be either `None` or the `logging.getLogger()` call below to pass no parameters.  This would get the root logger which seems appropriate in the case of the cli scripts.  But, I certainly may be missing something.

Reply:
The idea was to provide the library user with possibility to *only* configure 'canmatrix' logger tree without actually knowing the logger names. 
But the cli convert/compare scripts should probably call `setup_logger(None)`
I'll try to investigate more on common usage in libs.